pakyow-support/lib/pakyow/support/defineable.rb

```diff
@@ -61,6 +82,12 @@ def freeze
         super
       end
 
+      # Provide default make method
+      #
+      def make(*args, &block)
```

Comment:
Man, I keep misleading you 😑

So `stateful` registers the types of state that can be defined on a definable object. State does not have to be a definable object, though. For example, `Pakyow::App` is a definable object, and `Pakyow::Router` is registered as state that can be defined. This makes it possible to do this:

```ruby
Pakyow::App.define do
  router do
    ...
  end
end
```

So anything registered as stateful should define `make` at the class level that returns the configured state (which can be _either_ a class or an instance). We could still automatically mixin a default `make` method, but we would have to take a different approach.